vue/components/ui/molecules/table/table.vue

```diff
@@ -24,7 +45,28 @@
             <template v-for="(item, index) in sortedItems">
                 <slot name="before-row" v-bind:item="item" v-bind:index="index" />
                 <slot name="row" v-bind:item="item" v-bind:index="index">
-                    <tr v-bind:key="item.id" v-on:click="onClick(item, index)">
+                    <tr
+                        v-bind:class="[
+                            {
+                                selected: isRowSelected(item.id),
+                                highlighted: index === highlightedIndex
+                            }
+                        ]"
+                        v-bind:key="item.id"
+                        v-on:click.exact="onRowClick(item, index)"
+                        v-on:click.ctrl.exact="onRowCtrlClick(index, item.id)"
+                        v-on:click.meta.exact="onRowCtrlClick(index, item.id)"
+                        v-on:click.shift.exact="onRowShiftClick(index, item.id)"
+                    >
+                        <td class="checkbox-item" v-if="enableCheckboxes">
+                            <checkbox
+                                v-bind:size="8"
+                                v-bind:checked.sync="checkedItemsData[item.id]"

```

Comment:
v-bind:checked="checkedItemsData[item.id]"
v-on:update:checked="value => checkedItemsData[item.id] = value"
v-on:update:checked="value => $set(checkedItemsData, item.id, value)" <-----

Reply:
Since you're here, check ALL the `v-bind:SOMETHING.sync` for similar issues

Reply:
@gcandal it is working because of the watch has deep as true (see below). We are also using watch this way in other components too, ex.: in "filter-ripe".

```
checkedItemsData: {
    immediate: true,
    deep: true,
    handler: function(value) {
        this.selectionChange();
        this.$emit("update:checked-items", value);
    }
}
```

Reply:
I will use update either way, because when trying to uncheck all, `this.checkedItemsData = {};` the reactivity makes the the checkbox set the value to false.